packages/app/src/components/VariationSelector/VariationSelector.tsx

```diff
@@ -467,15 +453,17 @@ export function createVariationSelector(
     const result = await requestModal<RespondType>({
       class: ui.modalNoScroll,
       content: ({ respond }) => (
-        <ChangeHistoryContextProvider value={history}>
-          <ShowVariationSelector
-            currentVar={currentVar}
-            currentFlame={currentFlame}
-            transformId={tid}
-            variationId={vid}
-            respond={respond}
-          />
-        </ChangeHistoryContextProvider>
+        <Root adapterOptions={{ powerPreference: 'high-performance' }}>
```

Comment:
Are we nesting Root in a Root here?

Reply:
Uh oh, does seem like a duplicate, will remove the other one!
Nice catch